pmm/aws-staging-start.groovy

```diff
@@ -304,11 +304,14 @@ pipeline {
                         sudo yum -y update --security
                         sudo yum -y install https://repo.percona.com/yum/percona-release-0.1-7.noarch.rpm
```

Comment:
I would suggest updating this package to our latest version. Currently it is 1.0-25